manager/controllers/app/blueprint_controller.go

```diff
@@ -273,7 +281,10 @@ func (r *BlueprintReconciler) updateModuleState(blueprint *app.Blueprint, instan
 	blueprint.Status.ModulesState[instanceName] = state
 }
 
-func (r *BlueprintReconciler) reconcile(ctx context.Context, log logr.Logger, blueprint *app.Blueprint) (ctrl.Result, error) {
+func (r *BlueprintReconciler) reconcile(ctx context.Context, log zerolog.Logger, blueprint *app.Blueprint) (ctrl.Result, error) {
+	uuid := blueprint.GetAnnotations()[utils.FYBRIKAPPUUID] // used for logging
+	alog := log.With().Str(logging.CONTROLLER, "Blueprint").Str(logging.FYBRIKAPPUUID, uuid).Str("blueprint", blueprint.GetName()).Logger()
```

Comment:
Same comment. log seems to already include this information. Not clear why alog is needed.

Reply:
OK, changing